### PR TITLE
Add `Project Loved` Discord servers articles

### PR DESCRIPTION
**Pull request by aceticke**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

resolves some tasks on #6904

### CODE REVIEW

File: wiki/Community/Discord_servers/Project_Loved_Taiko/img/banner.jpg
Comment (Walavouchey) [inline wiki/Community/Discord_servers/Project_Loved_Taiko/img/banner.jpg:1]: use the server icons, not the banners

Comment (aceticke) [reply wiki/Community/Discord_servers/Project_Loved_Taiko/img/banner.jpg:1]: The server icons are in hilariously poor quality, unless someone has the full size images, maybe cc @Tanza3D

Comment (Tanza3D) [reply wiki/Community/Discord_servers/Project_Loved_Taiko/img/banner.jpg:1]: don't think i did those im afraid

**Review by TicClick — APPROVED** [review]

**Comment by aceticke** [comment]
I think Walavouchey wanted to shift to using the server icons instead which I have yet to implement, though your own thoughts are nice too @TicClick

**Comment by TicClick** [comment]
well I don't really have any issues with that, hence the approval. couldn't figure out if you guys gave up on making the icons and wanted it merged as-is or not

**Comment by peppy** [comment]
Because of course we need *more* discord servers.

**Review by cl8n — CHANGES_REQUESTED** [review]
the user lists in here seem needlessly verbose cuz it's just the captains and whoever else working on loved decided to be in the server at the time. I think it'd be better to explain that + link back to the project loved contributors article, rather than maintain separate lists

**Comment by aceticke** [comment]
> the user lists in here seem needlessly verbose cuz it's just the captains and whoever else working on loved decided to be in the server at the time. I think it'd be better to explain that + link back to the project loved contributors article, rather than maintain separate lists

It's important, at least for Discord server articles, to know who has administrative permissions in the server which is what I've done here, but I could always just condense it. Removing it would defeat the purpose of the list themselves especially because some servers operate differently and do not give permissions to all loved team members.

**Comment by cl8n** [comment]
all of that feels very unimportant to me / I can't imagine a way for a reader to find it useful... 

also surprised to learn from these articles that i am a "moderator" of some of these servers where I almost never visit in the first place lol. I get some sense of attributing more meaning to roles on discord than is really intended

File: wiki/Community/Discord_servers/Project_Loved_Taiko/en.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
...838763) (aceticke), ::{ flag=US }:: [Blushing](https://osu.ppy.sh/users/5927823) (blooshing),    ::{ flag=NL }:: [**Boaz**](https://osu.ppy.sh/users/13302996) (13oaz), ::{ flag=US }:: [clayton](https://osu.ppy.sh/users/3666350) (gp_clayton), ::{ flag=JP }:: [**Eriha**](https://osu.ppy.sh/users/16320311) (4yuni), ::{ flag=EC }:: [**Gamelan4**](https://osu.ppy.sh/users/9856910) (gamelan4), ::{ flag=US }:: [clayton](https://osu.ppy.sh/users/3666350) (gp_clayton), ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) (hivie), ::{ flag=LT }:: [huu](https://osu.ppy.sh/users/6044237) (huuishuu), ::{ flag=GB }:: [**Horiiizon**](https://osu.ppy.sh/users/8071438) (horiiizon), ::{ flag=ES }:: [kadoen](https://osu.ppy.sh/users/12780575) (deranged_cultist), ::{ flag=MY }:: [**ler1211**](https://osu.ppy.sh/users/19901680) (ler1211), ::{ flag=SG }:: [megumic](https://osu.ppy.sh/users/7537133) (megumic),  ::{ flag=KR }:: [POCARI SWEAT](https://osu.ppy.sh/users/5082685) (poca_p0ca), ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099) (radar), ::{ flag=BR }:: [**Sebola**](https://osu.ppy.sh/users/15821708) (sebola3461), ::{ flag=AT }:: [Yasuho](https://osu.ppy.sh/users/8458835) (yasuho) |
```
Comment (cl8n) [inline wiki/Community/Discord_servers/Project_Loved_Taiko/en.md:24]: this one's got extra spaces before boaz and my name duplicated, maybe some copy-paste error?

Comment (aceticke) [reply wiki/Community/Discord_servers/Project_Loved_Taiko/en.md:24]: yep will fix that

**Comment by kadoen** [comment]
I agree with clayton the list seems unnecessarily long, especially as it will shift when people enter/leave loved which will make these articles out of date relatively quickly. In mania server the mods are just the captains so there's not much point in duplicating the list there anyway.

**Comment by agatemosu** [comment]
like https://github.com/ppy/osu-wiki/pull/9745#issuecomment-1648641370, you should also put the @

**Comment by TicClick** [comment]
closing due to https://github.com/ppy/osu-wiki/issues/9934 -- feel free to rehash the descriptions and add them to the index page directly when the rework is done

**Pull request closed without merging**